src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/X86/X86Base.cs

```diff
@@ -84,5 +97,27 @@ public static unsafe (int Eax, int Ebx, int Ecx, int Edx) CpuId(int functionId,
         ///   PAUSE
         /// </summary>
         public static void Pause() => Pause();
+
+        /// <summary>
+        /// unsigned _udiv64(unsigned __int64 dividend, unsigned divisor, unsigned* remainder)
+        ///   DIV reg/m32
+        /// </summary>
+        public static (uint Quotient, uint Remainder) DivRem(uint lower, uint upper, uint divisor) => DivRem(lower, upper, divisor);
```

Comment:
nit: Keeping these alphabetized helps with finding the right API in these otherwise very large files, so I'd expect it to appear above `Pause`